implementations/rust/ockam/ockam_api/src/cli_state/traits.rs

```diff
@@ -41,7 +41,7 @@ pub trait StateDirTrait: Sized + Send + Sync {
     }
 
     /// Do not run any migration by default
-    async fn migrate(&self, _item_path: &Path) -> Result<()> {
+    async fn migrate(&self, _path: &Path) -> Result<()> {
         Ok(())
```

Comment:
I was expecting a migrate call here

Reply:
This is the default implementation, which does nothing. You have to add an specific impementation for the config structs you want to migrate.